src/core/a-animation.js

```diff
@@ -463,3 +463,21 @@ function strToBool (str) {
 function boolToNum (bool) {
   return bool ? 1 : 0;
 }
+
+/**
+ * A mapping of attribute to handle component attributes and singular attributes
+ *
+ * @param {object} element to look up attribute on
+ * @param {string} attr dot notation or singular 'color' or 'material.color'
+ * @returns {object|string|number} resulting value of type of component property.
+ */
+function getComputedAttributeFor (el, attribute) {
```

Comment:
Yeah, though it seems the purpose of it is to support dot notation, with fallback. Is that right?